### PR TITLE
Updated the QuizView to SwiftUI

### PR DESCRIPTION
**Pull request by below**

### CODE REVIEW

File: QuizList/QuestionView.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                            self.showCorrectAnswer = nil
+                            self.item = self.quizFactory.nextQuestion()
+                        }
+                    }
+                }, label: {
+                    Text(answerSet.answers[i]).highlight(i == showCorrectAnswer)
+                }).frame(maxWidth: .infinity)
+
+            }
+        }.font(.system(size: 32)).multilineTextAlignment(.center)
```
Comment (dasdom) [inline QuizList/QuestionView.swift]: I usually see this split up in several lines.

File: QuizList/QuestionView.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            return self.foregroundColor(.red)
+        } else {
+            return self
+        }
+    }
+}
+
+struct QuestionView: View {
+    var list: QuizList
+    var quizFactory: QuestionManufactory
```
Comment (dasdom) [inline QuizList/QuestionView.swift]: Looks like list and quizFactory can be made private.

File: QuizList/QuestionView.swift [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    var body: some View {
+        VStack(spacing: 10) {
+            Text("Item # \(list[item].number)").bold()
+
+            let answerSet = try! self.quizFactory.answers(question: self.item, number: 4)
+
+            ForEach(0..<answerSet.answers.count) { i in
+                Button(action: {
+                    if i == answerSet.correctAnswer {
+                        quizFactory.appendCorrectAnswer(i)
```
Comment (dasdom) [inline QuizList/QuestionView.swift]: Why are you appending the position of the correct answer?

Comment (below) [reply QuizList/QuestionView.swift]: What is wrong here is that I should append `self.item`. This is done to repeat items that were wrong

File: QuizList/QuestionView.swift (hunk 2 above)
Comment (below) [reply QuizList/QuestionView.swift]: I always wonder why. Recently, I ran into problems because someone aggressively made things private …

**Pull request merged**